fdbserver/VersionedBTree.actor.cpp

```diff
@@ -2586,10 +2588,11 @@ class DWALPager final : public IPager2 {
 	                               Standalone<VectorRef<PhysicalPageID>> pageIDs,
 	                               Reference<ArenaPage> page,
 	                               bool header = false) {
-		debug_printf("DWALPager(%s) op=%s %s ptr=%p\n",
+		debug_printf("DWALPager(%s) op=%s %s %d ptr=%p\n",
```

Comment:
```suggestion
		debug_printf("DWALPager(%s) op=%s %s encodingType=%d ptr=%p\n",
```
This new output field needs to be labeled, as the others are including the `%s` because the `toString()` scheme include labels.

Reply:
will update in #9020 .